### ISSUE TITLE
Is this code for a publication paper?

### ISSUE THREAD

**Issue by ithieund** [Shougong/bert-transformer-pointer-generator-networks#1]
Hello @Shougong , can you tell me if this code is for a publication paper or just an NLP project?
If you has a published paper for this code, please give me the paper name also.

Thank you so much!

**Comment by Shougong** [comment]
Hi, this code is just an NLP project.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年12月11日(星期六) 上午8:48
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: [Shougong/bert-transformer-pointer-generator-networks] Is this code for a publication paper? (Issue #1)





 
Hello @Shougong , can you tell me if this code is for a publication paper or just an NLP project?
 If you has a published paper for this code, please give me the paper name also.
 
Thank you so much!
 
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by ithieund** [comment]
Thank you for sharing.

**Comment by hzp1991** [comment]
Epoch / Total epochs: 31 / 200
100%|██████████| 568/568 [11:33<00:00,  1.22s/it]INFO - 12/15/21 16:02:38 - 6:46:34 - Train Epoch: 31, avg loss: 9.9029, accuracy: 6.18%
INFO - 12/15/21 16:03:56 - 6:47:51 - Validation. Epoch: 31, avg dev loss: 9.9011, accuracy: 6.23%
INFO - 12/15/21 16:03:56 - 6:47:51 - New best loss, Model saved

Epoch / Total epochs: 32 / 200
100%|██████████| 568/568 [11:35<00:00,  1.22s/it]INFO - 12/15/21 16:15:32 - 6:59:27 - Train Epoch: 32, avg loss: 9.9028, accuracy: 6.17%
INFO - 12/15/21 16:16:49 - 7:00:44 - Validation. Epoch: 32, avg dev loss: 9.9011, accuracy: 6.21%

Epoch / Total epochs: 33 / 200
100%|██████████| 568/568 [11:35<00:00,  1.22s/it]INFO - 12/15/21 16:28:24 - 7:12:20 - Train Epoch: 33, avg loss: 9.9029, accuracy: 6.17%
INFO - 12/15/21 16:29:42 - 7:13:37 - Validation. Epoch: 33, avg dev loss: 9.9011, accuracy: 6.23%

这准确率提不上来啊，loss也基本不动，或者变化很小。

**Comment by hzp1991** [comment]
> 

除了bert换成了chinese-roberta-wwm-ext，其他的都是默认参数。

**Comment by Shougong** [comment]
可能我最终调好的版本 没有上传hhhh



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年12月15日(星期三) 下午4:42
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Shougong/bert-transformer-pointer-generator-networks] Is this code for a publication paper? (Issue #1)





 
Epoch / Total epochs: 31 / 200
 100%|██████████| 568/568 [11:33<00:00,  1.22s/it]INFO - 12/15/21 16:02:38 - 6:46:34 - Train Epoch: 31, avg loss: 9.9029, accuracy: 6.18%
 INFO - 12/15/21 16:03:56 - 6:47:51 - Validation. Epoch: 31, avg dev loss: 9.9011, accuracy: 6.23%
 INFO - 12/15/21 16:03:56 - 6:47:51 - New best loss, Model saved
 
Epoch / Total epochs: 32 / 200
 100%|██████████| 568/568 [11:35<00:00,  1.22s/it]INFO - 12/15/21 16:15:32 - 6:59:27 - Train Epoch: 32, avg loss: 9.9028, accuracy: 6.17%
 INFO - 12/15/21 16:16:49 - 7:00:44 - Validation. Epoch: 32, avg dev loss: 9.9011, accuracy: 6.21%
 
Epoch / Total epochs: 33 / 200
 100%|██████████| 568/568 [11:35<00:00,  1.22s/it]INFO - 12/15/21 16:28:24 - 7:12:20 - Train Epoch: 33, avg loss: 9.9029, accuracy: 6.17%
 INFO - 12/15/21 16:29:42 - 7:13:37 - Validation. Epoch: 33, avg dev loss: 9.9011, accuracy: 6.23%
 
这准确率提不上来啊，loss也基本不动，或者变化很小。
 
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by ithieund** [comment]
@Shougong ,
I follow your code and the training loss doesn't decrease. Can you help to resolve this problem or can you share your latest working code?

Thank you so much!
